packages/modules/imodel-browser/src/containers/iModelGrid/IModelGrid.tsx

```diff
@@ -61,10 +82,15 @@ export interface IModelGridProps {
    */
   postProcessCallback?: (
     iModels: IModelFull[],
-    fetchStatus: DataStatus | undefined
+    fetchStatus: DataStatus | undefined,
+    searchText: string | undefined
   ) => IModelFull[];
   /**Component to show when there is no iModel */
   emptyStateComponent?: React.ReactNode;
+  /**  Create IModel Button component */
+  createIModelButton?: React.ReactNode;
+  /** flag to show search toolbar component along with view */
+  showSearchToolbar?: boolean;
```

Comment:
Do not include the top bar in this component, each application will want its own way of providing the search, what is displayed at the top of the grid, and how they want to decide if they should be in tile/list mode.

Please add instead a `viewMode?` prop that defaults to `tileView` but allow `listView` as well.

Please add a `search?` `string` prop that will be the `searchText` source, (naming it `search` will allow us to support a search object later if the API evolves to support search in description for example, but dont bother with that for now, when a string is provided, we'll search names.